src/propcache/__init__.py

```diff
@@ -1,6 +1,30 @@
 """propcache: An accelerated property cache for Python classes."""
 
-__version__ = "1.0.0.dev0"
+from typing import TYPE_CHECKING, List
+
+__version__ = "0.2.0.dev0"
 
 # Imports have moved to `propcache.api` in 1.0.0+.
-__all__ = ()
+# This module is now a facade for the API.
+if TYPE_CHECKING:
+    from .api import cached_property, under_cached_property
```

Comment:
Low type coverage in tests can probably be improved with `from .api import cached_property as cached_property, under_cached_property as under_cached_property` here.

Reply:
applied as https://github.com/aio-libs/propcache/pull/32/commits/a4242dce2ddb14f05f7eabff6c8f787c5eab9e00